docs/tutorials/benchmarking/README.md

```diff
@@ -0,0 +1,19 @@
+# Benchmarking in Syne Tune
+
+
```

Comment:
Benchmarking in machine learning (ML) refers to the evaluation and comparison of ML methods regarding their ability to learn patterns in ‘benchmark’ datasets. In Hyperparamter Optimization (HPO) domain, benchmarking compares a range of HPO methods on one or more tuning problems.

Reply:
(Please lets add above description first for first time readers) Then we follow up with:
This tutorial provides an overview of tooling that facilitates benchmarking of HPO algorithms in Syne Tune.